packages/trader/src/App/Components/Elements/ContractDrawer/market-closed-contract-overlay.jsx

```diff
@@ -2,32 +2,14 @@ import classNames from 'classnames';
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Text } from '@deriv/components';
-import { Localize } from '@deriv/translations';
-// eslint-disable-next-line import/no-useless-path-segments
-import MarketCountdownTimer from '../market-countdown-timer.jsx';
 
-const MarketClosedContractOverlay = ({ symbol }) => {
-    const [is_timer_loading, setIsTimerLoading] = React.useState(true);
-
-    return (
-        <div
-            className={classNames('contract-card__market-closed', {
```

Comment:
Loading timer was removed because `trading_times` BE Api call is removed? 

Also not seeing `trading_times` response map/render removed from contract details. Were we using them in contract details ?

Reply:
Yes, the call was used only to show the remaining timer which is calculated on FE based on trading_times response, there was no other usage of the response on the page. The idea is to get rid of this extra call and just show a new validation error from proposal API response instead.